docs/logging-install.md

```diff
@@ -0,0 +1,148 @@
+**This document discusses setting up of logging infrastructure in the OpenShift
+cluster. It assumes that you have an OpenShift 3.9.0 cluster already setup.**
+
+ If you don't have an OpenShift 3.9.0 cluster setup already, you can refer the
+[cluster-upgrade](cluster-upgrade.md) to *install* OpenShift 3.9.0 cluster. To
+install the cluster, just replace the `ansible-playbook` command with below:
```

Comment:
*To install OpenShift cluster

add OpenShift to be explicit?

As in earlier statement, we are asking to refer a document and later adding command for same (which is actually good), but just hinting for being explicit. WDYT?

Reply:
I didn't add because it's already mentioned in the previous statement that we're talking about OpenShift cluster.

Reply:
alright.